### PR TITLE
[W5.11][T16-A1]Qiu Haoze

### PR DESCRIPTION
**Pull request by qiuhaohao**
Add sort command, update user guide and test accordingly.

### CODE REVIEW

**Comment by qiuhaohao** [comment]
It seems that the check has failed because the help message is updated.

File: src/seedu/addressbook/data/person/UniquePersonList.java [hunk 1]
```diff
@@ -140,4 +140,12 @@ public boolean equals(Object other) {
                 || (other instanceof UniquePersonList // instanceof handles nulls
                         && this.internalList.equals(((UniquePersonList) other).internalList));
     }
+
+    /**
+     * Sorts all persons in the list according to their full name
+     */
+    public void sort(){
+        internalList.sort((person1, person2) -> (person1.getName().fullName.toLowerCase().compareTo(
```
Comment (madsonic) [inline src/seedu/addressbook/data/person/UniquePersonList.java:148]: is `.fullName` necessary?

File: src/seedu/addressbook/data/person/UniquePersonList.java [hunk 2]
```diff
@@ -140,4 +140,12 @@ public boolean equals(Object other) {
                 || (other instanceof UniquePersonList // instanceof handles nulls
                         && this.internalList.equals(((UniquePersonList) other).internalList));
     }
+
+    /**
+     * Sorts all persons in the list according to their full name
+     */
+    public void sort(){
```
Comment (madsonic) [inline src/seedu/addressbook/data/person/UniquePersonList.java:147]: coding style violation. [read this](https://oss-generic.github.io/process/codingStandards/CodingStandard-Java.html)

File: src/seedu/addressbook/commands/SortCommand.java [hunk 3]
```diff
@@ -0,0 +1,22 @@
+package seedu.addressbook.commands;
+
+import seedu.addressbook.data.person.ReadOnlyPerson;
+import java.util.List;
+
+/**
+ * Sorts entries in the address book according to first name.
+ */
+
+public class SortCommand extends Command{
```
Comment (madsonic) [inline src/seedu/addressbook/commands/SortCommand.java:10]: coding style violation. [read this](https://oss-generic.github.io/process/codingStandards/CodingStandard-Java.html)

**Comment by madsonic** [comment]
late submission.

good job. you are missing tests for this new feature though. Tests cases can illustrate specifications for your enhancements as well as force you to think about any assumptions that you might have made.

you are right about why the test is failing. So how do you fix it?

**Pull request closed without merging**